Dockerfile

```diff
@@ -19,16 +19,35 @@ WORKDIR node-red-contrib-security-nodes
 RUN npm install && npm link agile-sdk && npm link
 WORKDIR ..
 
-COPY agile-node-red-nodes agile-node-red-nodes
-RUN npm install agile-node-red-nodes
+ARG NODE=agile-node-red-nodes
+COPY $NODE $NODE
+RUN npm install $NODE
 
 # adding Xively support
-COPY node-red-contrib-agile-xively node-red-contrib-agile-xively
-RUN npm install node-red-contrib-agile-xively
+ARG NODE=node-red-contrib-agile-xively 
+COPY $NODE $NODE
```

Comment:
It looks this COPY does not have effect in the container. I enter the container and node-red-contrib-agile-xively is not there. Could it be that only the first ARG NODE=... is considered?